pypesto/profile/walk_along_profile.py

```diff
@@ -0,0 +1,118 @@
+import logging
+import numpy as np
+from typing import Callable
+
+from ..objective.constants import GRAD
+from ..optimize import Optimizer, OptimizerResult
+from ..problem import Problem
+from .result import ProfilerResult
+from .options import ProfileOptions
+
+logger = logging.getLogger(__name__)
+
+
+def walk_along_profile(
```

Comment:
Were changes made to this function? Fine if not

Reply:
no changes were made as far as i remember